Makefile.am

```diff
@@ -5577,6 +5577,11 @@ if BUILD_SELINUX
 endif
 endif
 
+CHILD_CAPABILITIES="cap_chown,cap_dac_override,cap_setuid,cap_setgid=ep"
+-setcap $(CHILD_CAPABILITIES) $(DESTDIR)$(sssdlibexecdir)/ldap_child
+-setcap $(CHILD_CAPABILITIES) $(DESTDIR)$(sssdlibexecdir)/krb5_child
+-setcap $(CHILD_CAPABILITIES) $(DESTDIR)$(sssdlibexecdir)/selinux_child
```

Comment:
selinux_child is optional I think. I don't have  that on my systems

Reply:
Wouldn't it be better to update code inside
https://github.com/SSSD/sssd/blob/fa9f6882bc5181edc404ebedf1ddaf5c92b917a5/Makefile.am#L5569
?

Reply:
I mean, capabilities should be instead of suid bit.

Reply:
Hi,

yes, fixed in the latest version.

bye,
Sumit